core-ui/src/main/res/values/nav_ids.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="tab_category" type="id" />
```

Comment:
I'm not sure that this id should be in core-ui module. Please remove it. If you need this id, I think this file should be in shop-ui module